### ISSUE TITLE
install job build command ignores working_directory parameter 

### ISSUE THREAD

**Issue by rbabayoff** [cypress-io/circleci-orb#315]
- what version of the orb are you currently using? 

1.26.0

- paste the `circle.yml` file if possible or at least the relevant portion:

```
workflows:
  test:
    jobs:
      - cypress/install:
          executor: node-12
          working_directory: packages/myapp1
          build: cd packages/myapp1 && npm run build
```

Note that I need to cd into my app's folder. Expected behavior should be to run `npm run build` in same folder as working_directory.

**Comment by stevesuh** [comment]
I think what's missing is setting working directory in the build step cypress/install command of the orb.
https://circleci.com/docs/2.0/configuration-reference/#run

```
# Install Job that can be performed before running jobs
  # often used to have a single installation step, and then
  # multiple run jobs can share the same pre-installed workspace.
  install:
    parameters:
      executor:
        type: executor
        default: base-10
        description: Cypress or custom executor name to use to run the install.
      post-checkout:
        description: Additional commands to run after performing Git checkout
        type: steps
        default: []
      post-install:
        description: |
          Additional commands to run after running install
          but before verifying Cypress and saving cache.
        type: steps
        default: []
      build:
        type: string
        default: ''
        description: |
          Custom build command to run after install to build the web application or web server.
      cache-key:
        description: Npm cache key
        type: string
        default: 'cache-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}'
      yarn:
        description: Use yarn to install NPM modules instead of npm.
        type: boolean
        default: false
      install-command:
        description: Overrides the default NPM or Yarn command
        type: string
        default: ''
      verify-command:
        description: Overrides the default NPM or Yarn command to verify Cypress
        type: string
        default: 'npx cypress verify'
      working_directory:
        description: |
          Directory containing package.json. Use this parameter if you're using a monorepo and your
          cypress tests aren't at the root of the repository (eg. `frontend`). Additionally,
          monorepo users will want to provide a `cache-key` parameter to key the cache with an
          appropriate checksum. (i.e. `cache-key: 'cache-{{ arch }}-{{ .Branch }}-{{ checksum "frontend/package.json" }}'`)
        type: string
        default: ''
    description: |
      Checks out code, installs dependencies, attaches code to the workspace for the future jobs
      to use (usually `cypress/run` follows the install step).
    executor: <<parameters.executor>>
    # be explicit about setting cache folder
    # https://github.com/cypress-io/circleci-orb/issues/98
    environment:
      CYPRESS_CACHE_FOLDER: ~/.cache/Cypress
    steps:
      - checkout
      - steps: << parameters.post-checkout >>
      - when:
          condition: << parameters.build >>
          steps:
            - install:
                install-command: << parameters.install-command >>
                verify-command: << parameters.verify-command >>
                yarn: << parameters.yarn >>
                post-install: << parameters.post-install >>
                build:
                  - run:
                      name: 'Build'
                      command: << parameters.build >>
                      working_directory: <<parameters.working_directory>>
                working_directory: << parameters.working_directory >>
                cache-key: << parameters.cache-key >>
      - unless:
          condition: << parameters.build >>
          steps:
            - install:
                install-command: << parameters.install-command >>
                verify-command: << parameters.verify-command >>
                yarn: << parameters.yarn >>
                post-install: << parameters.post-install >>
                working_directory: << parameters.working_directory >>
                cache-key: << parameters.cache-key >>
```

**Comment by stevesuh** [comment]
I am using 1.26.0 of the orb as well.

**Comment by bahmutov** [comment]
Released in 1.27.0
